README.md

```diff
@@ -2,23 +2,42 @@
 
 [![Build Status](https://travis-ci.org/allegro/ralph-cli.svg?branch=develop)](https://travis-ci.org/allegro/ralph-cli)
 [![Coverage Status](https://coveralls.io/repos/github/allegro/ralph-cli/badge.svg?branch=develop)](https://coveralls.io/github/allegro/ralph-cli?branch=develop)
+[![Documentation Status](https://readthedocs.org/projects/ralph-cli/badge/?version=latest)](http://ralph-cli.readthedocs.io/en/latest/?badge=latest)
 
-A command-line interface for Ralph.
+`ralph-cli` is a command-line interface for [Ralph][ralph]. At this moment, its
```

Comment:
> this is ralph-cli, cli interface to ralph, but it has nothing to offer for you

I didn't write anything like this :wink: We are offering one command - which is just the beginning - and that's what I want to emphasize *at this moment*.

> we could describe idea of scan "subcommand" briefly and mention about features-working-right-now [...]

I'm against putting too much info into this README, especially if there are dedicated sections for these two things (`scan` in "Concepts" and upcoming features in "Ideas for Future Development").